commands/netobserv

```diff
@@ -76,7 +81,9 @@ ${K8S_CLI_BIN} run \
 
 ${K8S_CLI_BIN} wait \
   -n netobserv-cli \
-  --for=condition=Ready pod/collector
+  --for=condition=Ready pod/collector || exit 1
```

Comment:
why did u drop `exit 1` here ?

Reply:
if the pods are not available, I want the script to crash here